PlanetFourAuthors/index.html

```diff
@@ -0,0 +1,65 @@
+<!doctype html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7" lang=""> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8" lang=""> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9" lang=""> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js" lang=""> <!--<![endif]-->
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+        <title>Planet Four: Archive</title>
+        <meta name="description" content="">
```

Comment:
Just one comment about these two lines. 

**Line 9:** Are we still happy with the title being "Planet Four: Archive?" We could also change it to "Planet Four: Authors," which may be more specific.

**Line 10:** I believe this line is used for search engine optimization, so we should probably give it more content than an empty string. Usually we use "Archived Zooniverse Project: Planet Four," but we could even be a bit more detailed and use "Archived Zooniverse Project: Planet Four contributing authors," or something similar.

Reply:
A couple of comments on the HTML, also not blocking.

- the `lang` attribute on the `html` tag can't be an empty string. `lang="en"` (or `lang="en-US"`) will tell user agents the page is in English. Search engines use this attribute when indexing pages. Screen-readers use it too.
- the conditional comments that add extra CSS classes for older versions of IE probably aren't needed any more.

Reply:
Made changes to both of these -- thanks!

Reply:
Added lang="en" to html tag as Jim suggested; kept conditionals as is (no harm no foul?).